packages/end-to-end/cypress/support/utilities.js

```diff
@@ -0,0 +1,45 @@
+/*
```

Comment:
What got timeboxed here was writing this function in ts and accepting an array instead of a single additional file -- after wrangling with the `cy.fixture()` promises for a while I left it be. It would be good for those improvements to be made someday 🙂